src/main/resources/com/arcbees/website/client/resources/css/developers.gss

```diff
@@ -158,7 +158,7 @@ ul.social {
 .slideshow a.prev::before,
 .slideshow a.next::before {
     content: "\e631";
-    font-family: "icons";
+    font-family: "icomoonregular";
```

Comment:
had to change it to make it work in firefox... we'll find a way to rename it somehow